invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -113,6 +113,7 @@ export class RDMDepositForm extends Component {
         },
         identifiers: {
           ...this.config.vocabularies.identifiers,
+
```

Comment:
that's weird :)

Reply:
Leftover from a rebase :sweat_smile: . Thanks!